src/runtime/runtime.js

```diff
@@ -1071,6 +1075,14 @@ export class ConfiguredRuntime {
     return this.eventManager_;
   }
 
+  /**
+   * Get the last offers flow.
+   * @return {?OffersFlow}
+   */
+  getLastOffersFlow() {
+    return this.lastOffersFlow_;
```

Comment:
Could you add test coverage to this line?

Reply:
Codecov says it isn't covered currently: https://app.codecov.io/gh/subscriptions-project/swg-js/compare/1669/diff